docs/reference/setup/install/targz-daemon.asciidoc

```diff
@@ -8,6 +8,10 @@ the process ID in a file using the `-p` option:
 ./bin/elasticsearch -d -p pid
 --------------------------------------------
 
+If you have configured secure settings and password-protected the {es}
+keystore, you will be prompted to enter the keystore's password before
+the program enters daemon mode.
```

Comment:
> ... before the program enters daemon mode.

Is there a reason for this qualifier?  If not, I'd omit it and use the same wording as in the other pages.

Reply:
You're right. It's not really necessary. I only put it in because at one point there was another way of doing things for daemon-mode — but no one needs to know about that.